### DISCUSSION TITLE
delay between jobs

### DISCUSSION THREAD

**Discussion by AmA-sud** [swar/Swar-Chia-Plot-Manager#360] [Q&A]
can a delay in starting the jobs be implemented?

it will be faster and easier for copying the plots on SATA drives after the finish of the plotting.
so at least all the jobs will not bottleneck each other in the last phase

**Comment by Berni-k** [comment]
I just fixed this problem in push https://github.com/swar/Swar-Chia-Plot-Manager/pull/380
